.github/ISSUE_TEMPLATE/request-for-position.yml

```diff
@@ -0,0 +1,49 @@
+name: Request for Mozilla Position on an Emerging Web Specification
+body:
+  - type: input
+    id: spec-title
+    attributes:
+      label: "Specification title"
+    validations:
+      required: true
+  - type: input
+    id: spec-url
+    attributes:
+      label: "Specification or proposal URL (if available)"
+      placeholder: "https://"
+  - type: input
+    id: spec-explainer
+    attributes:
+      label: "Explainer URL (if available)"
+      placeholder: "https://"
+  - type: input
+    id: spec-authors
+    attributes:
+      label: "Proposal author(s) (`@`-mention GitHub accounts)"
+      placeholder: "@handle, @another"
+    validations:
+      required: true
+  - type: input
+    id: caniuse-url
+    attributes:
+      label: "Caniuse.com URL"
+      placeholder: "https://"
```

Comment:
Should we move to MDN instead?

Reply:
I've no particular opinions on this, just porting the deprecated template to this new format.

Reply:
MDN is https://github.com/mozilla/standards-positions/pull/1146